docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version '3':
+
+services:
+  application:
+    container_name: template-node-ts
+    image: node:alpine
+    command: sh -c "npm install && npm run start:dev"
+    working_dir: /app
+    volumes:
+      - ./:/app
```

Comment:
This would avoid creating a volume because we don't want any data to be persisted between executions :)

Reply:
I putted the volume mirroring to inside /app container folder to run hot reload successfully on local environment